### PR TITLE
fix #77970 cgtips.org

### LINKED ISSUE #77970: sketchup.cgtips.org
### Issue URL (Anti Adblock Script)
 [https://sketchup.cgtips.org/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsketchup.cgtips.org%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by wout-codes**
#77970

Anti-AdBlock here too `cgtips.org`

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -515,11 +515,11 @@ variatkowo.pl,freegogpcgames.com,omgpu.com,arcadepunks.com#%#//scriptlet("set-co
...
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=googlesyndication-adsbygoogle,domain=dailyvideoreports.net|codesnail.com|ma-x.org|sitarchive.com|onlyfanesse.com|codingshiksha.com|graphicux.com|w3layouts.com|musike.net|windowsmatters.com|sonixgvn.net|freeiptvlists.com|avoiderrors.com|gatcha.org|sayrodigital.com|justandroid.net|4horlover.com|topnewsshow.com|3dzip.org|ovagames.com
+cgtips.org,dailyvideoreports.net,codesnail.com,ma-x.org,sitarchive.com,onlyfanesse.com,codingshiksha.com,graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet('prevent-addEventListener', 'load', 'daadb_get_data')
+cgtips.org,dailyvideoreports.net,codesnail.com,ma-x.org,sitarchive.com,onlyfanesse.com,codingshiksha.com,graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet("set-constant", "adsbygoogle.length", "undefined")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:519]: These rules aren't working.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:519]: They are working on my end.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:519]: I am not sure why. Then what do you recommend?

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:519]: > Just remove `cgtips.org` from there.

https://github.com/AdguardTeam/AdguardFilters/pull/77983#discussion_r598564312

**Review by zloyden — APPROVED** [review]

**Pull request merged**